roles/openshift-applier/tasks/process-file.yml

```diff
@@ -0,0 +1,39 @@
+---
+
+- name: "Determine location for the 'file' (if applicable)"
+  vars:
+    path: "{{ file }}"
+  import_tasks: check-file-location.yml
+  when:
```

Comment:
No need for this `when` here as that's already tested when this file is included in the first place.

Reply:
FYI, this is copy-paste work. These were all here before. I have cleaned them up now.

Reply:
So these were needed before, but not after you moved them. So, yes, fully understand that it was copy/paste, but this PR is changing the flow and hence not needed anymore - hence the request for a change.